src/JSON.jl

```diff
@@ -6,7 +6,7 @@ using Compat
 
 import Base.colon
 
-export json # returns a compact (or indented) JSON representation as a string
+export json, @json_str, @json_mstr   # returns a compact (or indented) JSON representation as a string
```

Comment:
this comment no longer describes the whole line; it should be moved to docstring anyways